Campaigns/apt ta17 293a ps.txt

```diff
@@ -0,0 +1,6 @@
+// Original Sigma Rule: https://github.com/Neo23x0/sigma/blob/master/rules/apt/apt_ta17_293a_ps.yml
```

Comment:
Maybe for documentation and clarity, add the description as a comment?
What do you think?

`// Detects renamed SysInternals tool execution with a binary named ps.exe as used by Dragonfly APT group and documentied in TA17-293A report`
then:
// Original Sigma Rule: https://github.com/Neo23x0/sigma/blob/master/rules/apt/apt_ta17_293a_ps.yml

Reply:
Good idea - I will add this (also to the others) ... but not today ;-)